src/packages/migrate/src/__tests__/fixtures/seed-sqlite/README.md

```diff
@@ -0,0 +1,11 @@
+# Seed example
+
+- `npm install`
+
+- Edit the `.env.example` with your PostgreSQL url
+
+- Rename the `.env.example` file to `.env`
+
+- Use `db push` to create the database structure `npx prisma db push --preview-feature`
+
+- Run the seed to populate the database `npm run seed`
```

Comment:
Should we not be using the prisma db seed command here?

Reply:
I forgot to remove that README actually, it's only for testing